src/styles/partials/components/_summary-list.scss

```diff
@@ -32,21 +28,16 @@
     p {
       color: $gray;
     }
-
-    & > div {
-      width: 75%;
-    }
   }
 
   h3 {
     color: $black;
   }
 
   i {
-    display: block;
     text-align: center;
     font-size: $h1-font-size;
-    width: 25%;
+    margin-right: $default-padding;
```

Comment:
this is  agood example why we should have one value for spacing margn / padding. which i think should have been 15px. That would be a big refactor now, so we can leave this as is.